devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -69,7 +69,10 @@ def parse(String description) {
     def powerDiv = 1
     def energyDiv = 100
 
-    if (event) {
+    if (event?.name == "switch" && description?.startsWith("catchall: ")) {

```

Comment:
I think the code would be a little simpler if you left the `if (event)` as is and put this additional check within the `if` block. 

Also I'd recommend doing this instead of looking for any `catchall` message:

```
Map descMap = zigbee.parseDescriptionAsMap(description)
if (event.name == "switch" && !descMap.isClusterSpecific descMap.commandInt == 0x0B)
```

That explicitly looks for the Default Response message.